src/components/graph/menus/create-node-menu.js

```diff
@@ -90,6 +103,29 @@ const CreateNodeMenu = ({
             action: () => createNetworkModificationNode('AFTER'),
             id: 'insertNetworkModificationNodeBelow',
         },
+        COPY_MODIFICATION: {
+            onRoot: false,
+            action: () => copyNetworkModificationNode(),
+            id: 'copyNetworkModificationNode',
+        },
+        PASTE_MODIFICATION_NODE_BEFORE: {
+            onRoot: false,
+            action: () => pasteNetworkModificationNode('BEFORE'),
+            id: 'pasteNetworkModificationNodeAbove',
+            disabled: selectedNodeForCopy == null,
+        },
+        PASTE_MODIFICATION_NODE_AFTER: {
+            onRoot: true,
+            action: () => pasteNetworkModificationNode('AFTER'),
+            id: 'pasteNetworkModificationNodeBelow',
+            disabled: selectedNodeForCopy == null,
+        },
+        PASTE_MODIFICATION_NODE_ON_NEW_BRANCH: {
+            onRoot: false,
+            action: () => pasteNetworkModificationNode('NEW_BRANCH'),
```

Comment:
Same remark in study-server PR : 
CHILD and NEW_BRANCH seems to have same signification ...
maybe remove CHILD ?

Reply:
will do that at the end to make sure it doesn't break anything since it was here before my PR

Reply:
fixed